index.js

```diff
@@ -229,9 +244,62 @@ var publicAgent = {
   private_: function() { return agent; }
 };
 
-module.exports = global._google_trace_agent = publicAgent;
+/**
+ * <p class="notice">
+ *   *This module is experimental, and should be used by early adopters. This
+ *   module uses APIs that may be undocumented and subject to change without
+ *   notice.*
+ * </p>
+ *
+ * This module provides Stackdriver Trace support for Node.js applications.
+ * [Stackdriver Trace](https://cloud.google.com/cloud-trace/) is a feature of
+ * [Google Cloud Platform](https://cloud.google.com/) that collects latency
+ * data (traces) from your applications and displays it in near real-time in
+ * the [Google Cloud Console][cloud-console].
+ *
+ * @constructor
+ * @alias module:trace
+ *
+ * @resource [What is Stackdriver Trace]{@link
+ *   https://cloud.google.com/cloud-trace/}
+ *
+ * @param {object} options - [Configuration object](#/docs)
+ */
+function Trace(options) {
```

Comment:
This constructor can be removed in a follow on PR as per the API design discussion.